### ISSUE TITLE
Missing docker images for 2.5.2 and 2.5.3

### ISSUE THREAD

**Issue by Lotte1990** [tensorflow/tensorflow#58958]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Source

source

### Tensorflow Version

2.5.2 and 2.5.3

### Custom Code

No

### Current Behaviour?

```shell
The docker images for 2.5.2 and 2.5.3 are missing here:
https://hub.docker.com/r/tensorflow/tensorflow
How can I obtain these images? I need the GPU images.
```

### Standalone code to reproduce the issue

```shell
sudo docker pull tensorflow/tensorflow:2.5.2-gpu
sudo docker pull tensorflow/tensorflow:2.5.3-gpu
```

### Relevant log output

_No response_</details>

**Comment by SuryanarayanaY** [comment]
Hi @Lotte1990 ,

Could you please refer to the link [here](https://hub.docker.com/r/tensorflow/tensorflow/tags/) where there is complete list of  images and let us know if it is helpful ?

Thank you!

**Comment by Lotte1990** [comment]
@SuryanarayanaY The docker images are not listed in the link you provided. 2.5.0 and 2.5.1 are present, but 2.5.2 and 2.5.3 are missing. Please add these docker images. I cannot upgrade beyond these versions because my models behave differently from 2.6 onwards.

**Comment by SuryanarayanaY** [comment]
@Lotte1990 ,
The issue put up forward. Could you please confirm whether you used the images 2.5.2 or 2.5.3 earlier.Whether they exists earlier ? I couldn't get track of those images.

**Comment by sachinprasadhs** [comment]
Hi @Lotte1990 , Usually when there are not much difference between `2.5.1 to 2.5.2 & 2.5.3`, these images will not be uploaded in docker hub, you can use `2.5.1` or you can use the `2.6.0` version, if you are looking for specific changes made in `2.5.2 & 2.5.3`.
Refer the similar issue [here](https://github.com/tensorflow/tensorflow/issues/54122#issuecomment-1024754940). Thanks!

**Comment by Lotte1990** [comment]
@sachinprasadhs I cannot upgrade to 2.6.0 and beyond because my models behave differently and I do need the changes made in 2.5.2 and 2.5.3. Please upload these versions.

**Comment by sachinprasadhs** [comment]
@angerson , @learning-to-play , Could you please look into this issue. Thanks!

**Comment by Lotte1990** [comment]
@angerson @learning-to-play @sachinprasadhs Any updates on this?

**Comment by qhaas** [comment]
We too are reliant upon TF 2.5.3.  There is a bug that breaks our stack in 2.5.1 that was fixed by 2.5.3 and our model does not work with 2.6+.  Please post a TF 2.5.3 container and help users avoid a `pip install tensorflow==2.5.3` upgrade to a possibly (less optimized) pypi package.

**Comment by MichaelHudgins** [comment]
Unfortunately 2.5 is significantly out of our current range of support.  It is unlikely we would create new containers for it.
